governance.md

```diff
@@ -0,0 +1,393 @@
+    requests. They are the primary mechanism of guiding the code review process
+    and merging pull requests.
+-   Make decisions about the Services that are run by The Project and manage
+    those Services for the benefit of the Project and Community.
+-   Make decisions when regular community discussion doesn’t produce consensus
+    on an issue in a reasonable time frame.
+
+### Council membership
+
+To become eligible for being a Steering Council Member an individual should
+meet some of the following criteria:
+
+-  Be a Project Contributor who has produced contributions that are substantial in
+quality and quantity
+-  Sustain this development consistently over several months
+-  Support the community through various activities (some examples below):
+   - code review
+   - answering user questions
+   - triaging bug reports
+   - participating constructively in broader conversations
+   - contributing to documentation
+   - maintaining infrastructure
+-  Demonstrate breadth by supporting the community outside of their particular
+   work interests or sub-project
+-  Be civil in public discourse
+
+Potential Council Members are nominated by existing Council members and voted
+upon by the existing Council after asking if the potential Member is interested
+and willing to serve in that capacity. The Council will be initially formed
+from the set of existing Core Developers who, as of 2019, have ownership rights
+over the organization.
+
+If a Council member becomes inactive in the project for a period of one year,
+they will be considered for removal from the Council. Before removal, the
+inactive Member will be approached to see if they plan to return
+to active participation. If not, they will be removed immediately upon a Council
+vote. If they plan to return to active participation soon, they will be
+given a grace period of one year. If they don’t return to active participation
+within that time period they will be removed by vote of the Council without
+further grace period. All former Council members can be considered for
+membership again at any time in the future, like any other Project Contributor.
+Retired Council members will be listed on the project website, acknowledging
+the period during which they were active in the Council.
+An inactive BDFL may also be ejected by a consensus of current members under
+these same conditions.
+
+The Council reserves the right to eject current Members, other than the BDFL,
+if they are deemed to be actively harmful to the project’s well-being, and
+attempts at communication and conflict resolution have failed. This action
+should not be taken lightly. The proposer must provide overwhelming
+justification as to why this action is needed (e.g. the Member violated the
+Code of Conduct) and not other technical reasons (e.g. we have different
+viewpoints on project direction) or personal reasons (e.g. they belong to a
+competitor's organization). We ask that a majority of owners respond for
```

Comment:
Hmm... what are "owners"?

Reply:
I guess owners of the org. Perhaps we should probably change it to steering council. Would you be comfortable pushing a change here, @pitrou?

Reply:
Thanks. Have changed to Steering Council members.